charts/pelorus/values.yaml

```diff
@@ -5,6 +5,11 @@
 # to reset password: htpasswd -s -b -n internal changeme
 openshift_prometheus_htpasswd_auth: internal:{SHA}+pvrmeQCmtWmYVOZ57uuITVghrM=
 openshift_prometheus_basic_auth_pass: changeme
+
+# Automatically set OpenShift's in-built prometheus stack as a federation point so Pelorus can use OpenShift metrics in Dashboards
+federate_openshift_monitoring:
```

Comment:
What if my cluster wasn't yet configured to enable cluster monitoring stack?

Reply:
Then this won't work, which is why its disabled by default.